documentation/Working_on_Bot/Test_Cases.md

```diff
@@ -95,3 +95,14 @@
    1. `.team "spaces #1" "musical voice channels" "Billy, Bob, Megan, Cary" "notAHexCode"`
    1. `.reportwin`
    1. `.reportwin "team 1337" "Jeffrey, Harry, Noble, Ali"`
+## Ban
```

Comment:
I am going to take advantage of this PR and make you also fill in the test cases for `Mod` command while you are waiting for me to merge my own PR 🙃

Reply:
done